bot/controllers/verify.js

```diff
@@ -80,31 +80,13 @@ class VerifyController extends BaseController {
           message.reply(verificationTimeout);
         }
       });
-      // Set up Nodemailer to send emails through gmail
-      const sendVerifyCode = nodemailer.createTransport({
-        service: 'gmail',
-        auth: {
-          user: process.env.EMAIL_USERNAME,
-          pass: process.env.EMAIL_PASS,
-        },
-      });
-      // Nodemailer email recipient & message
-      // TODO: Build email template
-      const mailOptions = {
-        from: process.env.EMAIL_USERNAME,
-        to: email,
-        subject: 'Armada Verification Code',
-        html: `<table><tr><td><p>Enter the code below into Discord, in the same channel on the Armada Server. Verification will timeout after ${(timeoutInMiliseconds / 1000) / 60} minutes from first entering the !verify command.</p></td></tr><tr><td><h2>Verification Code: ${code}</h2></td></tr></table>`,
-      };
-      // Call sendMail on sendVerifyCode
-      // Pass mailOptions & callback function
-      sendVerifyCode.sendMail(mailOptions, (err, info) => {
-        const errorMsg = 'Oops, looks like the email can not be sent. It\'s not you, it\'s me. Please reach out to a moderator to help you verify.';
-        if (err) {
-          message.reply(errorMsg);
-          util.log('Email not sent', err, 3);
-        } else {
-          util.log('Email details', info, 3);
+      const emailType = 'verify';
+      const emailSubject = 'Armada Verification Code';
+      const emailBodyString = `<table><tr><td><p>Enter the code below into Discord, in the same channel on the Armada Server. Verification will timeout after ${(timeoutInMiliseconds / 1000) / 60} minutes from first entering the !verify command.</p></td></tr><tr><td><h2>Verification Code: ${code}</h2></td></tr></table>`;
+      sendEmail(message, email, emailSubject, emailBodyString, emailType, (sendStatus) => {
+        if (sendStatus) {
+          const numPointToAdd = 1;
```

Comment:
Question, could this be deleted? I'm not sure why points would be added in the verify file when all they are doing is becoming verified. This could probably be deleted.

Reply:
I thought that verifying could warrant the users first point. What did you think? I could take it out.

Reply:
Maybe take out that section and in the part where it creates the new member instead of setting it to 0 it could be set to a variable of 1. Just a thought since that part already sets it to zero to begin. We could have it start with 1 points instead of 0. It  does make sense to reward them with at least a point for verifying though, you're right.

Reply:
Yea, that makes more sense to keep it dry.